readeventsfile.go

```diff
@@ -55,17 +71,45 @@ func VerifiableEventsFromData(data []byte) ([]VerifiableEvent, error) {
 		return nil, err
 	}
 
-	var verifiableEvents []VerifiableEvent
-	for _, raw := range eventsJson {
-		ve, err := NewVerifiableEvent([]byte(raw))
-		if err != nil {
-			return nil, err
-		}
-		verifiableEvents = append(verifiableEvents, ve)
+	verifiableEvents, err := logverification.NewVerifiableEvents(eventsJson)
+	if err != nil {
+		return nil, err
 	}
+
 	return verifiableEvents, nil
 }
 
+// ReadDecodedEventsFromFile reads datatrails events from a file and returns a
+// normalized list of raw binary items.
+//
+// See EventListFromData for the content expectations (must be a list of events
+// or single event from datatrails api)
+func ReadDecodedEventsFromFile(fileName string) ([]logverification.DecodedEvent, error) {
```

Comment:
If we're reading events from a file, I think its kind of implied that they're decoded. Given that the function signature tells us its a DecodedEvent, I think we can drop this from the name.